js/src/components/help-popover/index.js

```diff
@@ -47,6 +47,7 @@ const HelpPopover = ( { id, children } ) => {
 				<Popover
 					focusOnMount="container"
 					onClose={ handlePopoverClose }
+					{ ...props }
```

Comment:
It's nice to forward props to `Popover` so that we can fully control it. And we could also handle `onClick` in [the same way in `AppButton`](https://github.com/woocommerce/google-listings-and-ads/blob/ef2ed8bd4c4ea721401026a53468eacfaba23bb0/js/src/components/app-button/index.js#L40-L50), thus, the following use would work smoothly.
```js
<HelpPopover
	onClose={ () => { /* do something */ } }
/>
```

Reply:
Thanks for the review @eason9487 

I'm not sure if I fully understand this. I'm passing actually the `...props` to the Popover, hence you can define the props you want.

I guess you suggested to add `onClick` and `onClose` callbacks in the `handleButtonClick` and `handlePopoverClose`,  but not 100% sure.

Reply:
> And we could also handle `onClick` in [...]

Sorry for my typo. 🙏  It should be `onClose` and I meant this processing:

```js
const HelpPopover = ( { id, onClose = noop, children, ...props } ) => {
	const [ showPopover, setShowPopover ] = useState( false );

	// ... omitted

	const handlePopoverClose = () => {
		setShowPopover( false );
		onClose();
	};

	// ... omitted
};
```